Extension/browser/chrome/lib/background-page.js

```diff
@@ -25,104 +25,91 @@ var browser = browser || chrome;
         BaseEvent.apply(this, arguments);
     };
 
-    TabEvent.prototype = {
-        __proto__: BaseEvent.prototype
-    };
+    Object.setPrototypeOf(TabEvent.prototype, BaseEvent.prototype);
```

Comment:
Guys on stackoverflow advise switching to standard inheritance instead of messing with __proto__:
http://stackoverflow.com/questions/7509831/alternative-for-the-deprecated-proto